### PR TITLE
warrior medals

### PR DESCRIPTION
**Pull request by ezio416**
warrior medals, some small formatting fixes

### CODE REVIEW

File: Render/Render.as
Comment (Banalian) [inline Render/Render.as:1]: out of curiosity, why the change here? I don't mind it, I just wonder what bug you could see when not using \u2212 directly

File: Controller/Main.as [hunk 1]
```diff
@@ -161,9 +161,12 @@ void RefreshLeaderboard(){
...
         // Insert all entries in our temporary entry array
         for(uint i = 0; i< resp.positions.Length; i++){
-            if(resp.positions[i].time == -1){
+            if (false
+                || resp.positions[i].time == -1
+                || (i > 0 && resp.positions[i].time == resp.positions[i - 1].time)
+            )
                 continue;
-            }
+
```
Comment (Banalian) [inline Controller/Main.as:169]: I assume the false is just to reorganize how that looks to align the other two conditions x), noice

What's the last condition preventing? (or to rephrase, what was the edge case you saw?)
I'm guessing you want to prevent the case where CM and WM have the same time ? (or sbville's medal and these two)

File: Render/RenderSettings.as [hunk 2]
```diff
@@ -2,7 +2,7 @@
 void RenderSettingsCustomization(){
 
     if(!UserCanUseThePlugin()){
-        UI::Text("You don't have the required permissions to use this plugin. You at least need the standard edition.");
+        UI::TextWrapped("You don't have the required permissions to use this plugin. You at least need the standard edition.");
```
Comment (Banalian) [inline Render/RenderSettings.as:5]: *The more you know*
Thanks a lot for this!

**Review by Banalian — APPROVED** [review]
No changes to requests, Thanks a lot for taking to time to add support for the medal in ELP :D! I'll approve it right away, but I'll still wait for the answers to my curiosity questions before merging and doing an update with it.

File: Controller/Main.as (hunk 1 above)
Comment (ezio416) [reply Controller/Main.as:169]: Yeah, I tend to format conditionals with a `false` like that to make it hopefully more readable but mainly so I can easily comment entire lines to test things. In this case I think because the second condition is somewhat long, it would be hard to read being all on one line. Also yes, this is to hide the time if there are duplicates (CM/WM, CM/SB, WM/SB). Without this, I noticed the time would show in the window twice but both entries would say "Champion" and I couldn't find another way around it.

File: Render/Render.as
Comment (ezio416) [reply Render/Render.as:1]: I believe it was a standard dash (`-`) before, which does not have the same width as a `+`, so I swapped it out for the special character so they line up more nicely

File: Render/RenderSettings.as (hunk 2 above)
Comment (ezio416) [reply Render/RenderSettings.as:5]: Of course! Now if only I could find a way to limit the width of input fields as `SetNextItemWidth` doesn't seem to do that for me...

File: Controller/Main.as (hunk 1 above)
Comment (ezio416) [reply Controller/Main.as:169]: Ideally I think I would want any duplicates to show so you can easily see if a CM is equal to a WM or whatever but with your logic I just couldn't find the way to do that

File: Render/Render.as
Comment (Banalian) [reply Render/Render.as:1]: Ahh, nice change then, thanks!

File: Controller/Main.as (hunk 1 above)
Comment (Banalian) [reply Controller/Main.as:169]: Yeah, I guess it will stay as-is for now, and I'll open myself an issue to look at it later, since it would be better to see all medals yeah

**Comment by Banalian** [comment]
Merged! Thanks again for your time, this is highly appreciated :D!
